src/vehicle_gui_base.h

```diff
@@ -76,14 +77,26 @@ struct BaseVehicleListWindow : public Window {
 		GB_END,
 	};
 
-	GroupBy grouping;                         ///< How we want to group the list.
-	VehicleList vehicles;                     ///< List of vehicles.  This is the buffer for `vehgroups` to point into; if this is structurally modified, `vehgroups` must be rebuilt.
-	GUIVehicleGroupList vehgroups;            ///< List of (groups of) vehicles.  This stores iterators of `vehicles`, and should be rebuilt if `vehicles` is structurally changed.
-	Listing *sorting;                         ///< Pointer to the vehicle type related sorting.
-	byte unitnumber_digits;                   ///< The number of digits of the highest unit number.
+	GroupBy grouping;                           ///< How we want to group the list.
```

Comment:
Absolutely nitpicking, but I don't think this needs to match the indentation below the enum (especially since the enum itself doesn't match). 😃

Reply:
I guess you were expecting a different change, but well... :P